src/routes/userRoutes.ts

```diff
@@ -26,7 +25,8 @@ userRoutes.put("/passwordupdate", isLoggedIn, validateSchema(passwordUpdateSchem
 userRoutes.post("/login", emailValidation,validateSchema(logInSchema),userLogin);
 userRoutes.post("/register", emailValidation, validateSchema(signUpSchema), createUserController);
 userRoutes.put("/passwordupdate", isLoggedIn, validateSchema(passwordUpdateSchema), updatePassword);
-userRoutes.post("/2fa-verify", isTokenFound, tokenVerification);
+userRoutes.get("/2fa-verify/:token",tokenVerification);
```

Comment:
are you verifying from get method?

Reply:
oooh I missed this

Reply:
again for that get its verification link that user click and u can't use post using link since it will be opened directly in browser
perhaps when we put fronted url its possibl

Reply:
this will be handled  from front-end tha means link will be front-end link, but leave it for now